aleph-client/src/contract/convertible_value.rs

```diff
@@ -193,3 +186,124 @@ where
         }
     }
 }
+
+impl<Elem: TryFrom<ConvertibleValue, Error = anyhow::Error>> TryFrom<ConvertibleValue>
+    for Vec<Elem>
+{
+    type Error = anyhow::Error;
+
+    fn try_from(value: ConvertibleValue) -> std::result::Result<Self, Self::Error> {
+        let seq = match value.0 {
+            Value::Seq(seq) => seq,
+            _ =>  bail!("Failed parsing `ConvertibleValue` to `Vec<T>`. Expected `Seq(_)` but instead got: {:?}", value),
+        };
+
+        let mut result = vec![];
+        for element in seq.elems() {
+            result.push(ConvertibleValue(element.clone()).try_into()?);
+        }
```

Comment:
Suggestion: something along the lines of

```suggestion
        seq.elems().into_iter().map(|x| ConvertibleValue(x).try_into()).collect()
```
? Not sure if better.

Reply:
meh, you would have to:
 - add `.clone()` to `x`
 - convert `Vec<Result<_>>` into `Result<Vec<_>>` through further `iter().collect()`
and it gets meh

generally, I find loop a better option when the mapper returns `Result` or `Option`